_docs/organizations-and-collections.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Organizations and Collections
+permalink: /docs/publisher-tutorials/organizations-and-collections/
+---
+
+# Organizations and Collections
+
+## Organizations
+Organizations are landing pages for collaborations, institutions, consortiums, companies, etc. that allow users to showcase tools and workflows. This is achieved through the creation of collections, which are groupings of related tools and workflows. The users of an organization do not need to own the tools or workflows in any way, the tools and workflows just have to be published. Collections can be thought of as a playlist on a Music Streaming Service where tools and workflows are analogous to individual songs. They can be shared publicly, and the user does not need to own them.
+
+### Creating an organization
+To create an organization request, go to the [organizations](https://dockstore.org/organizations) page and select `Create Organization Request`. Any user can request to create an organization by filling out the following form. For now, the request must be approved by a Dockstore curator in order to be public. Until it is, you are still able to edit it, add collections, add members, etc.
+
+![Create Organization Request](/assets/images/docs/CreateOrganizationRequest.png)
+
+The fields for name, display name, and topic are all required. These can be changed later.
+* **Name** - the name used in URLs and as an identifier (unique)
+* **Display Name** - the pretty name used anywhere the organization is mentioned that allows for other characters such as spaces (unique)
```

Comment:
answer in text: what does unique mean?

Reply:
I'll change to 'unique across Dockstore'?

Reply:
A little unclear, maybe unique across all organization names in Dockstore

Reply:
For both this and the previous line, I would recommend a full sentence, without parens instead of having `(unique)` in parens. E.g., "The organization name must be unique across all of Dockstore".

I say this as a person who tends to overuse parenthesis. :)